src/components/sidebar.jsx

```diff
@@ -225,6 +251,7 @@ const SideBar = ({ user, signOut, recentGroups }) => {
       <SideBarFriends user={user} />
       <SideBarArchive user={user} />
       <SideBarSearch user={user} />
+      <SideBarMemories />
```

Comment:
@kadmil According to the document "Memories" should be below the Bookmarklet section and above the Donate.